.travis.yml

```diff
@@ -45,6 +45,26 @@ matrix:
       compiler: gcc
       env: CMAKE_BUILD_TYPE=Release BACKEND="OPENMP"
 
+# Install newer CMake. The distribution comes with CMake 3.12.4 but we require at least 3.16
+install:
+  - CMAKE_VERSION=3.17.1
```

Comment:
Why not:

```
DISTRIB_CODENAME=$(cat /etc/lsb-release | grep DISTRIB_CODENAME | awk -F '=' '{print $NF}')
wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc | apt-key add -
apt-add-repository "deb https://apt.kitware.com/ubuntu/ ${DISTRIB_CODENAME} main"
```

Reply:
I just copied from our `Dockerfiles`. We could try your suggestions but it doesn't matter as long as we don't activate `Travis` again.